src/components/drawer/index.js

```diff
@@ -119,6 +127,10 @@ class NewDrawer extends PureBaseComponent {
         ? this.renderLeftActions
         : this.renderRightActions
       : undefined;
+
+    this.state = {
+      leftX: new Animated.Value(0)
+    };
```

Comment:
No need to keep values that don't affect the state in state (like animated values)
Also, the name is a little too general, since there's a lot of things that refer to left/right, it would help to have a better name that hint what's this animation is responsible for. (e.g leftActionX, leftContentX, etc..)

Reply:
Done